### ISSUE TITLE
project onboarding: Developer Experience SIG

### ISSUE THREAD

**Issue by svrnm** [open-telemetry/community#2189]
With https://github.com/open-telemetry/community/pull/2144 merge, the Developer Experience SIG can be onboarded, here are the TODOs to get them started:

- [ ] Create project board
- [ ] SIG Slack Channel
- [ ] Meeting Notes (Google Docs)
- [ ] Add Meeting Times, GC Liaison, Slack Channel, Google Doc to sigs.yml
- [ ] GitHub Teams
- [ ] Repository (?)

@tsloughter @austinlparker @tedsuo @lmolkova @dmathieu @julianocosta89 @martinjt @samsp-msft @stevejgordon

**Comment by svrnm** [comment]
Meeting Notes: https://docs.google.com/document/d/1QDZYAvGJbqgodJaTxRPmRZh-fEJjb6XstvPUst6h50w/edit

**Comment by tsloughter** [comment]
Thanks @svrnm !

**Comment by tsloughter** [comment]
Do each of these just require opening a community repo issue?

Aside from finding a meeting time. Which I guess I'll just use a regular doodle for.

**Comment by svrnm** [comment]
We can work on them here if possible but some might require a dedicated issue, eg the teams&repo setup

**Comment by tsloughter** [comment]
Created https://github.com/orgs/open-telemetry/projects/105/

@svrnm how do I make get it public?

**Comment by svrnm** [comment]
@open-telemetry/technical-committee @austinlparker @trask can one of you set the project board shared above to public? Thanks

**Comment by svrnm** [comment]
@tsloughter you should be able to make it public yourself now, can you double check (cc @danielgblanco)

**Comment by svrnm** [comment]
> @svrnm are you able to create a repo? We need one for hosting/tracking discussions.

No, I can not, we need to raise an issue for that (repo management), see https://github.com/open-telemetry/community/issues/2191 for an example. I can create that for you, or you just go ahead and do it
 
> I hope to update sig.yml soon but waiting participants final opinion on meeting time.

Sure! You could add it already without meeting hours, depending on how long it will take for that to be finalized

The slack channel still needs to be renamed?

**Comment by tsloughter** [comment]
@svrnm thanks, I went ahead and did it https://github.com/open-telemetry/community/issues/2242

**Comment by tsloughter** [comment]
Yea, still waiting on a slack channel rename. I'll poke people about that today.

**Comment by svrnm** [comment]
> Yea, still waiting on a slack channel rename. I'll poke people about that today.

Thanks

> @svrnm thanks, I went ahead and did it #2242

Thanks as well :-)

**Comment by austinlparker** [comment]
Created the NA meeting

**Comment by lmolkova** [comment]
Created repository and teams.

**Comment by svrnm** [comment]
thanks all, last thing missing is to also update the project file at https://github.com/open-telemetry/community/blob/main/projects/developer_experience.md
